cellcounter/cc_kapi/README.md

```diff
@@ -12,9 +12,42 @@ cc_kapi is listed in the INSTALLED_APPS in settings.py:
 In urls.py we have a declaration to point the keyboard api related requests to the cc_kapi urls.py
 
     urlpatterns = patterns('',
-                           url(r'^api/keyboard/', include('cellcounter.cc_kapi.urls')),
+                           url(r'^api/keyboards', include('cellcounter.cc_kapi.urls')),
                            )
 
 Finally, in deployment, we'll need to run ```collectstatic``` in order to pull the Javascript into the main static directory.
 
-    python manage.py collectstatic
\ No newline at end of file
+    python manage.py collectstatic
+
+
+# Usage
+
+Access through api/keyboards/
+
+api/keyboards/
+    GET: lists all available keyboards, including builtin and user keyboards for both desktop and mobile
+
+api/keyboards/desktop/
+    GET: lists all available desktop keyboards, including builtin and user keyboards
+    POST: adds a new desktop keyboard
+
+api/keyboards/desktop/<id>/
+    GET: gets the desktop keyboard with identifier <id> (<id>: db_key | 'builtin' | 'default')
+    PUT: updates the specified desktop keyboard
+    DELETE: deletes the specified desktop keyboard
+
+api/keyboards/desktop/<id>/set_default
+    POST: sets the specified desktop keyboard as the default desktop keyboard
```

Comment:
See also a comment below. I think there are two options here:

1. `PUT` to `api/keyboards/desktop/default` to update the default (fits with the idea of a single point of access to a given resource)
2. If not, is a `PUT` to `set_default` more appropriate, as a default will always exist (from the point of view of a resource), and you are modifying a reference to an existing keyboard